test/controllers/event_controller_test.exs

```diff
@@ -0,0 +1,20 @@
+defmodule Dwylbot.EventTestController do
+  use Dwylbot.ConnCase
+  alias Poison.Parser, as: PP
+
+  test "POST /event/new - no error", %{conn: conn} do
+    payload = "./test/fixtures/add_label.json"
+    |> File.read!()
+    |> PP.parse!()
```

Comment:
@nelsonic It's documented [here](https://github.com/christopheradams/elixir_style_guide#pipe-operator)

I've opened an issue for documenting omission of parens (and other syntax decisions) [here](https://github.com/dwyl/learn-elixir/issues/47)